packages/rabbitmq/src/rabbitmqTxSubmitProvider.ts

```diff
@@ -1,27 +1,62 @@
 import { Buffer } from 'buffer';
 import { Cardano, HealthCheckResponse, ProviderError, ProviderFailure, TxSubmitProvider } from '@cardano-sdk/core';
 import { Channel, Connection, connect } from 'amqplib';
+import { ErrorDescription, deserializeErrors } from './errorsSerialization';
 import { Logger, dummyLogger } from 'ts-log';
+import { TX_SUBMISSION_QUEUE, txBodyToId, waitForPending } from './utils';
 
-export const TX_SUBMISSION_QUEUE = 'cardano-tx-submit';
+const moduleName = 'RabbitMqTxSubmitProvider';
+
+/**
+ * Configuration options parameters for the RabbitMqTxSubmitProvider
+ */
+export interface RabbitMqTxSubmitProviderConfig {
+  /**
+   * Use the algorithm to get a dummy Tx Id from a Tx body. Used for tests
+   */
+  dummyTxId?: boolean;
```

Comment:
Is this really necessary? Could just build an actual tx :thinking:

Reply:
No, it is not strictly required; I evaluated it as useful.
There are lots of tests already using fake transactions, without we should rework them all; for new tests without it we are forced to always use a real tx.
From my point of view it's just a choice, but my experience in the team is not enough to take this decision.
Please explicitly ask me to remove it, and I'll remove it.

Reply:
Agree with @mkazlauskas. It would also be a lot better not to have "dummy" things outside of the test code itself

Reply:
removed

Reply:
A commit hash in a comment would help reviewing requested changes @iccicci